integration/test/Test/AssetUpload.hs

```diff
@@ -0,0 +1,18 @@
+module Test.AssetUpload where
+
+import API.BrigInternal
+import API.Cargohold
+import SetupHelpers
+import Testlib.Prelude
+
+testAssetUploadUnverifiedUser :: HasCallStack => App ()
+testAssetUploadUnverifiedUser = do
+  user <- randomUser OwnDomain $ def {activate = False}
+  bindResponse (uploadAsset user) $ \resp -> do
+    resp.status `shouldMatchInt` 403
```

Comment:
We should also test unknown user case since we also added code for that.

Reply:
Done and done